src/app/submit-item/components/submit-page/submit-page.component.html

```diff
@@ -25,8 +25,10 @@ <h1>Fall einreichen</h1>
         <div class="middle-section-container">
           <h2>Dein Fall</h2>
           <p>
-            Wenn es etwas gibt, was du gerne prüfen möchtest, trage es hier ein. Gibt es diesen Fall bereits, erhältst du das Ergebnis
-            sofort. Falls dein Fall neu ist, werden wir ihn sofort an unser Team von Detektiven weitergeben, die den Fall dann für dich
+            Wenn es etwas gibt, was du gerne prüfen möchtest, trage es hier ein. Gibt es diesen Fall bereits, erhältst
+            du das Ergebnis
+            sofort. Falls dein Fall neu ist, werden wir ihn sofort an unser Team von Detektiven weitergeben, die den
+            Fall dann für dich
             prüfen.
```

Comment:
"Team an Detektiven" -> "Detektiv-Team"
"_sofort_. Falls dein Fall neu ist, werden wir ihn _sofort_ ..."  would delete the second "sofort".
"Falls dein Fall neu ist, werden wir ihn an unser Detektiv-Team weitergeben, die den Fall dann für dich prüfen."